pkg/cloudprovider/providers/gce/gce_forwardingrule.go

```diff
@@ -99,3 +90,41 @@ func (gce *GCECloud) ListGlobalForwardingRules() (*compute.ForwardingRuleList, e
 	v, err := gce.service.GlobalForwardingRules.List(gce.projectID).Do()
 	return v, mc.Observe(err)
 }
+
+// GetRegionForwardingRule returns the RegionalForwardingRule by name & region.
+func (gce *GCECloud) GetRegionForwardingRule(name, region string) (*compute.ForwardingRule, error) {
```

Comment:
nit: s/GetRegionForwardingRule/GetRegionalForwardingRule

Global vs. Regional

Same for the rest of CRUD functions.